From f4129fcf88780b01c6fc02fdbda544c086c22159 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Thu, 27 Sep 2018 21:26:12 +0800
Subject: [PATCH] enlarge timeout of wait_for_modules

Workaround runqemu without kvm, starting dbus in it very slow

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 pyanaconda/startup_utils.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyanaconda/startup_utils.py b/pyanaconda/startup_utils.py
index cab54b4..2da4a2c 100644
--- a/pyanaconda/startup_utils.py
+++ b/pyanaconda/startup_utils.py
@@ -442,7 +442,7 @@ def set_installation_method_from_anaconda_options(anaconda, ksdata):
         log.error("Unknown method: %s", anaconda.methodstr)
 
 
-def wait_for_modules(timeout=60):
+def wait_for_modules(timeout=600):
     """Wait for the DBus modules.
 
     :param timeout: seconds to the timeout
-- 
2.7.4

